### PR TITLE
fix(Reactions): improve styling in popover

### LINKED ISSUE #12398: Styling of **And x other participant(s)**
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Write a message
2. Get reactions by >3 people

### Expected behaviour

Subtle UI

### Actual behaviour

image

^ indentation is strange. Bold text too.

## Talk app

**Talk app version:** (see apps administration page: `/index.php/settings/apps`)

**Custom Signaling server configured:** yes/no and version (see  Talk administration settings: `/index.php/index.php/settings/admin/talk#signaling_server`)

**Custom TURN server configured:** yes/no (see Talk administration settings: `/index.php/settings/admin/talk#turn_server`)

**Custom STUN server configured:** yes/no (see  Talk administration settings: `/index.php/settings/admin/talk#stun_server`)

## Browser

**Microphone available:** yes/no

**Camera available:** yes/no

**Operating system:** Windows/Ubuntu/Mac/...

**Browser name:** Firefox/Chrome/Safari/...

**Browser version:** 124/125/...

### Browser log

<details>
```
Insert your browser log here, this could for example include:
a) The javascript console log
b) The network log
c) ...
```

</details>

## Server configuration

**Operating system**: Ubuntu/RedHat/...

**Web server:** Apache/Nginx

**Database:** MySQL/Maria/SQLite/PostgreSQL

**PHP version:** 8.1/8.2/8.3

**Nextcloud Version:** (see administration page)

**List of activated apps:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your server installation folder
```
</details>

**Nextcloud configuration:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
```
</details>

### Server log (data/nextcloud.log)
<details>

```
Insert your server log here
```
</details>

### PR DESCRIPTION
**Pull request by DorraJaouad**
### ☑️ Resolves

* Fix #12398

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.vue [hunk 1]
```diff
@@ -256,7 +259,7 @@ export default {
 					summary.push(this.getDisplayNameForReaction(list[item]))
 				}
 			}
-			return summary.join(', ')
+			return summary.join(', ') + (this.reactionsCount(reaction) > 3 ? ', ' : '')
```
Comment (Antreesy) [inline src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.vue:262]: Comma before `and` in sentences like `First, second and third` is not used in:
- Russian
- British English
- German
- French (from internet)
- Arabic (from internet)
Exception: American English

I'd say, we drop this change? Or provide a complete translatable string for 1,2,3,4+ options?

**Review by Antreesy — COMMENTED** [review]
Addressed some changes. Apart from it, looks much nicer

File: src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.vue (hunk 1 above)
Comment (DorraJaouad) [reply src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.vue:262]: Let's drop it

**Review by Antreesy — APPROVED** [review]

**Pull request merged**